.github/workflows/benchmark-command.yml

```diff
@@ -25,12 +26,15 @@ on:
           - high
           - good
           - medium
+          - dev
```

Comment:
What does the dev means here in terms of preset? Do we have such preset supported in AG and bench?

Reply:
I think here it is the versions of AG.
Yes, we have such preset supported for timeseries.

Reply:
Yes

Reply:
I'd appreciate some clarification on the need for this new dev preset. Could you provide further information on why it is being introduced?

Reply:
It's because AG-Bench uses this preset for timeseries - https://github.com/autogluon/autogluon-bench/blob/master/sample_configs/amlb_configs/frameworks_example.yaml
And hence adding dev will make it pick up the version specified in the frameworks.yaml file